src/main/scala/outwatch/dom/package.scala

```diff
@@ -1,13 +1,15 @@
 package outwatch
 
 import cats.effect.IO
+import org.scalajs.dom.Element
 
 package object dom extends Implicits with ManagedSubscriptions with SideEffects with MonixOps {
 
   val Sink = outwatch.Sink
   val Handler = outwatch.Handler
 
-  type VNode = IO[VTree]
+  type VNodeT[+T <: Element] = IO[VTree[T]]
```

Comment:
I don't think I like the `VNodeT` name. It uses the same convention as Monad transformers in cats, which might confuse done people familiar with them 😁

Reply:
True, that name is not really intuitive. Do you have a better name? :)

Reply:
@LukaJCB @fdietze What would be a good name for `VNodeT`?

Reply:
`TypedVNode`?

Reply:
`GenericVNode`?